Mod2.Lection1.Hw1/Mod2.Lection1.Hw1/Logger.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System.Text;
+
+namespace Mod2.Lection1.Hw1
+{
+    internal class Logger
+    {
+        private static readonly Logger instance = new();
+
+        static Logger() { }
+        private Logger() { }
+
+        readonly List<string> loggers = new();
+
+        internal static Logger GetInstance => instance;
+
+        internal void Error(string message)
```

Comment:
It was supposed to be a generic Log method, that would accept the type of the log (Error) and message. That implementation works, but a bit different from what I wanted to see =)